stories/Typography/Components.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Why did you disable this rule?

Reply:
Lint was accousing this and blocking the test